dotnet/src/SemanticKernel.Skills/Skills.Memory.VectorDB/Skills.Memory.VectorDB.csproj

```diff
@@ -0,0 +1,34 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+        <RepoRoot>$([System.IO.Path]::GetDirectoryName($([MSBuild]::GetPathOfFileAbove('.gitignore', '$(MSBuildThisFileDirectory)'))))</RepoRoot>
+    </PropertyGroup>
+    <Import Project="$(RepoRoot)/dotnet/nuget/nuget-package.props" />
+
+    <PropertyGroup>
+        <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+        <AssemblyName>Microsoft.SemanticKernel.Skills.Memory.VectorDB</AssemblyName>
```

Comment:
Update all 3 mentions of `Microsoft.SemanticKernel.Skills.Memory.VectorDB`
to
`Microsoft.SemanticKernel.Skills.Memory.Qdrant`

Reply:
I had an IVectorDBClient interface that was for consistency across the various VectorDB clients like the initial Milvus implementation I was testing. I would like to do something similar to Langchain where their is a base across implementations. I had the namespace for that reason. I can take this out but wanted to share the rationale.